guides/common/assembly_installing-server-connected.adoc

```diff
@@ -25,7 +25,7 @@ Note that the {Project} installation script is based on Puppet, which means that
 ⁠
 To avoid this and determine which future changes apply, use the `--noop` argument when you run the installation script.
 This argument ensures that no actual changes are made.
-Potential changes are written to `/var/log/foreman-installer/{project-context}.log`.
+Potential changes are written to `{installer-log-file}`.
```

Comment:
I wonder whether this is correct for Orcharino? `project-context` is defined as `orcharino` there, but in other places of the docs you use `--scenario katello`.

Reply:
It def changes the rendering from `/var/log/foreman-installer/orcharhino.log` to `/var/log/foreman-installer/foreman.log`, and I bet the later is totally wrong, but is the former right? Or should it be `katello.log`?

cc @maximiliankolb

Reply:
We use both `/var/log/foreman-installer/foreman.log` and `/var/log/foreman-installer/katello.log` with much much more content in the Katello log... 🤷 

Thanks for adding an attribute for this! I think `katello.log` is the right call!

Reply:
*how* do you manage to get both? The installer should be very vocal not to do such things ;)

Or do you have setups with non-content proxies, that are deployed with `foreman-installer --scenario foreman --no-enable-foreman`?

Reply:
Well, if you have the `foreman` scenario present it does run the migrations and that generates logs IIRC, even if you only use the scenario `katello`.

Reply:
I was specifically checking orcharhino Server; not an orcharhino Proxy. To my knowledge, we use scenario `katello`.